src/akshita/MaxCharFrequency.java

```diff
@@ -0,0 +1,44 @@
+package akshita;
+
+public class MaxCharFrequency {
+	
+	int getMaxCharFrequncy(String input, char ch) {
+		int count =0;
+			for(int sindex=0;sindex<input.length();sindex++) {
+				if(input.charAt(sindex)>ch) {
```

Comment:
condition is incorrect, kindly try once.

Reply:
will update it